### ISSUE TITLE
Creating project with existing folder not raising error

### ISSUE THREAD

**Issue by goanpeca** [Anaconda-Platform/anaconda-project#107]
See https://github.com/ContinuumIO/navigator/issues/1195

**OS:** Windows, Mac
**Anaconda: 4.4.0**

**Steps to reproduce:**
- Install Anaconda.
- Set the path of project.
- Create some project.
- Create a new folder in that path.
![1a](https://cloud.githubusercontent.com/assets/27444898/26408756/1cf88d58-40b8-11e7-9728-e1ff99b85dcf.png)

- Now create new project with the same name as of folder.
![1c](https://cloud.githubusercontent.com/assets/27444898/26408808/3ccc4e08-40b8-11e7-92f6-37b688ed4cdc.png)
![1d](https://cloud.githubusercontent.com/assets/27444898/26408845/4ebb7bb6-40b8-11e7-85fc-1d6b0aaaa6c1.png)
- Click on any other project
![1e](https://cloud.githubusercontent.com/assets/27444898/26408873/6606b588-40b8-11e7-8c7c-0e25434d1530.png)

**Actual Result:**
New created project vanished 

**Expected Result:**
An error message must appear if user cannot make the project with the same name as existed folder.

**Comment by havocp** [comment]
see also https://github.com/Anaconda-Platform/anaconda-project/issues/53

**Comment by goanpeca** [comment]
@havocp yep its similar but on #53 you suggest doing nothing, here I need some error if I try to create a project by passing the path and that path exists (even if it is just an empty folder)

**Comment by mcg1969** [comment]
Thanks for opening this issue, and apologies for the long delay in responding.

Anaconda Project has not been actively developed in some time and has largely been superseded by newer alternatives:

* [pixi](https://pixi.sh/) and [uv](https://docs.astral.sh/uv/) represent the first truly successful successors.
* [conda-workspaces](https://github.com/conda-incubator/conda-workspaces) is a promising pixi-compatible alternative for users who prefer to stay closer to the conda ecosystem.

Going forward, work on this repository is focused on critical bug fixes and on tooling to help users migrate existing Anaconda Project projects to pixi (and, by extension, conda-workspaces). See the [Pixi support](https://anaconda-project.readthedocs.io/en/latest/pixi-support.html) page in our documentation for details on the `anaconda-project export-pixi` command, which can convert an existing `anaconda-project.yml` into a `pixi.toml` for you.

The `anaconda-project` package itself will remain available, so existing projects relying on its stable feature set can continue to be supported. However, we do not anticipate further enhancement work, and for that reason we are closing out the bulk of our open issue backlog — including this one.

If this issue describes a critical regression in the stable feature set (as opposed to an enhancement or an edge case in a feature we no longer plan to expand) and you have a minimal reproducer against a current release, please feel free to reopen and we will take another look.

Thank you for your interest in Anaconda Project.
